lib/LaTeXML/Package/TeX.pool.ltxml

```diff
@@ -898,7 +898,7 @@ DefParameterType('CSName', sub {
 
 DefMacro('\csname CSName', sub {
     my ($gullet, $token) = @_;
-    Let($token, '\relax') unless defined LookupMeaning($token);
+    $STATE->assignMeaning($token, $STATE->lookupMeaning(T_CS('\relax'))) unless defined LookupMeaning($token);
```

Comment:
Cool, I see you found a short-enough $STATE invocation! Looks very satisfying to me personally.